awx/main/scheduler/kubernetes.py

```diff
@@ -51,6 +52,72 @@ def list_active_jobs(self, instance_group):
 
         return pods
 
+    @classmethod
+    def create_secret(self, job):
+        task = collections.namedtuple('Task', 'id instance_group')(id='', instance_group=job.instance_group)
+        pm = PodManager(task)
+        registry_cred = job.execution_environment.credential
+        host = registry_cred.get_input('host').split('/')[0]
```

Comment:
Wouldn't someone commonly enter `http://my_host/` for the `host` field of the credential ? The split would return `http:` in this case.

Reply:
True, I'll use urlparse to just grab the host.

Reply:
Get the port too